src/python/pants/option/global_options.py

```diff
@@ -592,7 +592,7 @@ def register_bootstrap_options(cls, register):
             advanced=True,
             default=pants_version(),
             daemon=True,
-            help="Use this Pants version. Note that Pants only uses this to verify that you are "
+            help="Use this Pants version. Note that Pants only uses this to verify that you are "  # noqa: PANTSBIN
```

Comment:
Why?

Reply:
I was going to blame `flake8` but I think it's a bug with the line in the plugin. It'll be on line `600` (The AST visitor doesn't give us a way to flag these as different strings AFAICT, so I'll report on the last line)

Reply:
Although, i forgot I can just use `bin_name` here since we default

Reply:
(Also, FWIW we can't change this to `end_lineno` until we drop Python 3.7)